tachyon/math/finite_fields/prime_field_base.h

```diff
@@ -18,6 +18,8 @@ class PrimeFieldBase : public Field<F> {
  public:
   using Config = typename PrimeFieldTraits<F>::Config;
 
+  // An invariant of a field which is either a prime number or the number zero,
```

Comment:
I took this definition from
https://encyclopediaofmath.org/wiki/Characteristic_of_a_field

Characteristic is the smallest positive integer such that n*e(or 1, the unit element of the field)  = 0.
If no such n exists, then the characteristic of the field is defined to be zero.

But as you said, since we are only dealing with the Prime Field, there is no case for zero. I will remove the mention of zero.